src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    if (declaration.trim() !== '') {
+      const [key, value] = declaration.split(':');
+
+      styleObject[key.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE STYLE] - When chaining methods, ensure that they all have the same indentation to improve the readability of the code. In this case, you could chain the `split`, `filter`, and `forEach` (or `reduce` if you take the previous suggestion) with proper indentation.